config/phase-0/connectors/definitions/mongodb.json

```diff
@@ -0,0 +1,241 @@
+{
+	"_id": "MongoDB",
+	"connectorRef": {
+		"connectorHostRef": "mongodb",
+		"displayName": "MongoDB Connector",
+		"bundleVersion": "1.5.19.1",
+		"systemType": "provisioner.openicf",
+		"bundleName": "org.forgerock.openicf.connectors.mongodb-connector",
+		"connectorName": "org.forgerock.openicf.connectors.mongodb.MongoDBConnector"
+	},
+	"poolConfigOption": {
+		"maxObjects": 10,
+		"maxIdle": 10,
+		"maxWait": 150000,
+		"minEvictableIdleTimeMillis": 120000,
+		"minIdle": 1
+	},
+	"resultsHandlerConfig": {
+		"enableNormalizingResultsHandler": false,
+		"enableFilteredResultsHandler": false,
+		"enableCaseInsensitiveFilter": false,
+		"enableAttributesToGetSearchResultsHandler": true
+	},
+	"operationTimeout": {
+		"CREATE": -1,
+		"UPDATE": -1,
+		"DELETE": -1,
+		"TEST": -1,
+		"SCRIPT_ON_CONNECTOR": -1,
+		"SCRIPT_ON_RESOURCE": -1,
+		"GET": -1,
+		"RESOLVEUSERNAME": -1,
+		"AUTHENTICATE": -1,
+		"SEARCH": -1,
+		"VALIDATE": -1,
+		"SYNC": -1,
+		"SCHEMA": -1
+	},
+	"configurationProperties": {
+		"customSensitiveConfiguration": null,
+		"createScriptFileName": "Create.groovy",
+		"targetDirectory": null,
+		"customizerScriptFileName": null,
+		"warningLevel": "1",
+		"authenticateScriptFileName": null,
+		"scriptExtensions": "groovy",
+		"scriptOnResourceScriptFileName": null,
+		"minimumRecompilationInterval": "100",
+		"deleteScriptFileName": "Delete.groovy",
+		"scriptBaseClass": null,
+		"scriptRoots": [
+			"jar:file:connectors/mongodb-connector-1.5.19.1.jar\u0021/scripts/mongodb/usersRoles/"
```

Comment:
Been advised that this should be changed to `jar:file:connectors/mongodb-connector-1.5.19.1.jar\u0021/scripts/mongodb/` (NB connector still not working with this)

Reply:
Updated